hoomd/ComputeThermo.h

```diff
@@ -206,6 +206,93 @@ class PYBIND11_EXPORT ComputeThermo : public Compute
             return h_properties.data[thermo_index::potential_energy];
             }
 
+
+        // <----------------- Components of the pressure tensor --------------
+
+        Scalar getPressureXX()
```

Comment:
Use one method to return the pressure tensor. You can either write a C++ method that returns the pressure tensor as a python list [xx, xy, xz, yy, yz, zz] or you can export the `PressureTensor` struct to Python and build that list in the Python property after calling the existing `getPressureTensor` method.

Reply:
I think returning the Python list will be cleanest.

Reply:
Why use just one method to get pressure tensor components? Also, we still want to expose each component of the pressure tensor individually as its own loggable quantity, right?

Reply:
Because the pressure tensor is a single "object" that should be gettable with one name. The elements of that tensor are naturally accessed by index.